### DISCUSSION TITLE
Dictation Flag - Not functioning as documented.

### DISCUSSION THREAD

**Discussion by ghost** [deepgram/community#532] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I am working on a medical dictation application that streams live audio and I am looking at Deepgram for the speech recognition side of things. I am trying to use the dictation query param but it doesn't work as [documented](https://developers.deepgram.com/docs/smart-format#dictation). 

The documentation claims you can say things like, "period", but when I say, "period", the response I receive is the empty string ''. The only commands that work are new line and new paragraph except the document says new paragraph should return <\n> which is incorrect, but it functions correctly.

Am I missing something here or is there a newer version of the documentation for this function that I have missed?

Thank you.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?model=nova-2-medical&language=en-US&dictation=true&punctuate=true&smart_format=true&measurements=true&keywords=PILF%3A2

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

1ee7d997-be9e-4ceb-83d1-62dd5636d0d6

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by ghost** [comment]
If it helps I just threw in another example. Request ID: 686f540a-f72c-43b2-909b-027d67743e24. All I do is open the stream and dictate, "period", 5 times making sure to dictate it by itself with a detectable pause between each attempt. Each response shows this output with the time based parameters being updated of course.

`{
    "type": "Results",
    "channel_index": [
        0,
        1
    ],
    "duration": 3.0500002,
    "start": 12.79,
    "is_final": true,
    "speech_final": false,
    "channel": {
        "alternatives": [
            {
                "transcript": "",
                "confidence": 0,
                "words": []
            }
        ]
    },
    "metadata": {
        "request_id": "686f540a-f72c-43b2-909b-027d67743e24",
        "model_info": {
            "name": "2-medical-nova",
            "version": "2023-11-04.1144",
            "arch": "nova-2"
        },
        "model_uuid": "1ebdff47-c4c5-43c4-a40b-aeac58d15574"
    }
}`

As you can see the transcript is empty and the confidence is 0.

**Comment by ghost** [comment]
I have put in a fair bit of testing tonight trying to figure out what may be going on. I haven't found a good answer, but I do have some more details that may help. So the other query parameters appear to be working just fine and even when I set dictation = true I can use the punctuation commands as part of a sentence and they work. So for example if I say, "The patient presents today with a headache {period}". The resulting text is accurate, (e.g. "The patient presents today with a headache."). 

These punctuation commands only fail when they are on their own so to use the last example if I stated, "The patient presents today with a headache", then paused, and on its own stated, "period", the second result I get has no text and a confidence of 0 as shown in my second update.

**Comment by ghost** [comment]
I have noticed one other odd behavior. When I have an instance of this issue the result is delayed compared to other results. Not much, but you can definitely notice it is slower to return the 0 confidence, empty text result as compared to the lightning fast response for a sentence of text.

**Comment by ghost** [comment, marked as answer]
Ended up getting some traction in the community discord. Closing this thread.
